service/src/main/resources/application.yml

```diff
@@ -25,6 +25,9 @@ env:
     in-kubernetes: ${TERRA_COMMON_KUBERNETES_IN_KUBERNETES:false}
     pod-name: ${TERRA_COMMON_KUBERNETES_POD_NAME:}
     pod-name-filter: ${TERRA_COMMON_KUBERNETES_POD_NAME_FILTER:}
+  pipelines:
+    imputation:
+      storageWorkspaceStorageUrl: ${IMPUTATION_STORAGE_WORKSPACE_STORAGE_URL:https://lze96253b07f13c61ef712bb.blob.core.windows.net/sc-e426d483-dca5-45e3-b5a4-9e7e38126aed}
```

Comment:
can we add a comment to what workspace in what environment this default is pointing to

Reply:
good call, done!